prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```

Comment:
Since `flex-basis` only works on "flex items" we want to put this property on the rule set for the `.card-wrapper` instead of the `#card-deck`.

Reply:
<img width="1128" alt="2022-10-04 (7)" src="https://user-images.githubusercontent.com/54726911/193890582-910b8793-6169-4e4b-bcfd-e01afe1164dd.png">